draft-ietf-anima-brski-prm.md

```diff
@@ -1976,127 +2120,146 @@ The response has the Content-Type `application/jose+json`.
-The Content-Type header is kept as `application/jose+json` as depicted in the example in {{estat}}.
+According to {{Section 5.9.4 of !RFC8995}}, the registrar SHOULD respond with an HTTP 200 OK in the success case or MAY fail with an HTTP 404 client error status code.
+The registrar MAY use the response body to signal success/failure details to the service technician operating the Registrar-Agent.
 
-The registrar MUST verify the signature of the pledge enroll status.
-Also, the registrar SHALL validate that the pledge is an accepted device of the domain based on the contained product-serial-number in the LDevID certificate referenced in the header of the enroll status.
-The registrar SHOULD log this event.
-In case the pledge enroll status indicates a failure, the pledge was unable to verify the received LDevID certificate and therefore signed the enroll status with its IDevID credential.
-Note that the signature verification of the status information is an addition to the described handling in {{Section 5.9.4 of !RFC8995}}, and is replacing the pledges TLS client authentication by DevID credentials in {{!RFC8995}}.
+If the eStatus indicates failure, the registrar MAY decide that for security reasons the pledge is not allowed to reside in the domain.
+In this case, the registrar MUST revoke the pledge EE certificate.
+An example case for the registrar revoking the issued certificate is when the pledge was not able to verify the received EE certificate and therefore did not accept it for installation.
 
 
-### Response (no artifact)
+### Request Artifact: Enroll Status (eStatus)
 
-According to {{Section 5.9.4 of !RFC8995}}, the registrar SHOULD respond with an HTTP 200 OK in the success case or fail with HTTP 4xx/5xx status codes.
+Identical to the Enroll Status (eStatus) artifact received from the pledge as defined in {{estatus_artifact}}.
+The Registrar-Agent MUST NOT modify eStatus artifacts.
 
-Based on the failure case the registrar MAY decide that for security reasons the pledge is not allowed to reside in the domain. In this case the registrar MUST revoke the certificate.
-An example case for the registrar revoking the issued LDevID for the pledge is when the pledge was not able to verify the received LDevID certificate and therefore did send a 406 (Not Acceptable) response.
-In this case the registrar may revoke the LDevID certificate as the pledge did no accepted it for installation.
 
-The Registrar-Agent may use the response to signal success / failure to the service technician operating the Registrar-Agent.
-Within the server log the registrar SHOULD capture this telemetry information.
+### Response (no artifact)
+
+In this exchange, the response is a result of the HTTP(S) default transport for this specification.
+There is no artifact provided to the Registrar-Agent.
+
 
 
 ## Query Pledge Status {#query}
```

Comment:
This whole section is about a new mechanism and I reached it rather late.
Please review the complete new version of it.

Reply:
Just reviewed the section and committed only two editorial changes as "that MUST" appeared twice in the Status Trigger and Status Data description. The section reads fine from my understanding an aligns with the intend of the prior version.

Reply:
Pulled new version where you fixed this.